packages/sdk/README.md

```diff
@@ -12,42 +12,10 @@ yarn install @linear/sdk
 import { Linear } from "@linear/sdk";
 
 const linear = new Linear({
-  token: "<your developer key>"
+  apiKey: "<your developer key>",
```

Comment:
A small wording update to be more indicative to what the developer would see in the settings menu. Small things, but consistency really helps make tools approachable.